.buildkite/scripts/run-tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -euo pipefail
+
+# Pre install:
+source .buildkite/scripts/pre-install-command.sh
+add_bin_path
+with_mage
+with_go_junit_report
+
+set +e
+mage -debug test | tee tests-report-linux.txt
+exit_code=$?
```

Comment:
nit. Add after this block again `set -e` flag
```suggestion
set +e
mage -debug test | tee tests-report-linux.txt
exit_code=$?

set -e
```

That would help if `go-junit-report` command fails, so it could report that this step has failed too.